packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -507,12 +517,21 @@ class Actions extends StatefulWidget {
   /// Creates a dependency on the [Actions] widget that maps the bound action so
   /// that if the actions change, the context will be rebuilt and find the
   /// updated action.
-  static Action<T>? find<T extends Intent>(BuildContext context, {bool nullOk = false}) {
+  ///
+  /// The optional `intent` argument supplies the type of the intent to look for
+  /// if the concrete type of the intent sought isn't available. If not
+  /// supplied, then `T` is used. When using the `intent` argument, use `Intent`
+  /// as `T`.
```

Comment:
Does Dart actually force you to explicitly specify `T` in that case? I thought it would allow you to just call `find(context, intent: DoNothingIntent())`...

Reply:
It would, but our lint rules do not.

Reply:
I removed that sentence, since it doesn't apply to everyone.